Corvus.m

```diff
@@ -21,46 +21,71 @@ + (Corvus *)sharedInstance {
   return sharedInstance;
 }
 
+- (Stacktrace *)buildStacktrace:(DDLogMessage *)logMessage {
+  // NSArray<NSString *> *symbols = [NSThread callStackSymbols];
+  // TODO parse the actual symbols[c] string, if we're able to get the real ones
```

Comment:
In my opinion, it'd be great to send up the full stacktrace to Sentry (only for `DDLogError`, probably?).  Unfortunately, the Corvus logger is called by CocoaLumberjack on a worker thread, where the original call stack is no longer available.  

- I've got a question out to the CocoaLumberjack folks [here](https://github.com/CocoaLumberjack/CocoaLumberjack/issues/276) about this.
- Happy to remove the TODOs for now if you'd prefer, just lmk